src/init.cpp

```diff
@@ -2650,52 +2612,15 @@ bool initVideo()
 bool changeVideoMode(int new_xres, int new_yres)
 {
 	if (new_xres) {
-		xres = new_xres;
+		xres = std::max(1024, new_xres);
 	}
 	if (new_yres) {
-		yres = new_yres;
+		yres = std::max(720, new_yres);
 	}
 	printlog("changing video mode (%d x %d).\n", xres, yres);
-#ifdef PANDORA
-	GO_InitFBO();
-#else
-	int c;
-
-	// delete old texture names (they're going away anyway)
-	glDeleteTextures(MAXTEXTURES, texid);
 
 	// destroy gui fbo
-#ifndef EDITOR
 	Frame::fboDestroy();
-#endif
-
-	// delete vertex data
-	if ( !disablevbos )
-	{
-		for ( c = 0; c < nummodels; c++ )
-		{
-			SDL_glDeleteBuffers(1, &polymodels[c].vbo);
```

Comment:
Did any of these polymodels buffers need to be deleted?

Reply:
They were deleted previously because the gl context was being destroyed leaving them invalid.

They no longer need to be deleted.